### PR TITLE
prepare version 2.3

### PR DESCRIPTION
**Pull request by larskuhtz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # `chainweb-node` Changelog
 
+## 2.3 (2020-11-11)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-11-19T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2020-12-22T00:00:00Z**.
```
Comment (larskuhtz) [inline CHANGELOG.md:9]: @mightybyte @slpopejoy This is just a placeholder. The final value must be still filled in.

File: node/ChainwebNode.hs [hunk 2]
```diff
@@ -493,10 +492,10 @@ pkgInfoScopes =
 -- -------------------------------------------------------------------------- --
 -- main
 
--- KILLSWITCH for version 2.2
+-- KILLSWITCH for version 2.3
 --
 killSwitchDate :: Maybe String
-killSwitchDate = Just "2020-11-19T00:00:00Z"
+killSwitchDate = Just "2020-12-22T00:00:00Z"
```
Comment (larskuhtz) [inline node/ChainwebNode.hs:498]: @mightybyte @slpopejoy This is just a placeholder. The final value must be still filled in.

File: node/ChainwebNode.hs [hunk 3]
```diff
@@ -518,148 +517,3 @@ main = do
   where
     timeFormat = iso8601DateFormat (Just "%H:%M:%SZ")
 
--- -------------------------------------------------------------------------- --
--- Database Directory Migration
---
--- TODO: This code can be removed in chainweb-2.2.
```
Comment (larskuhtz) [inline node/ChainwebNode.hs:524]: This justifies the removal of this code.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
+
+*   Support for Pact Events (#1157)
+*   Upgrade Pact version 3.7 (#1158)
```
Comment (larskuhtz) [inline CHANGELOG.md:12]: TODO: this probably needs some more details

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,16 @@
 # `chainweb-node` Changelog
 
+## 2.3 (2020-11-11)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-11-19T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2020-12-22T00:00:00Z**.
```
Comment (larskuhtz) [inline CHANGELOG.md:9]:
```suggestion
This version will stop working on **2021-01-14T00:00:00Z**.
```

File: node/ChainwebNode.hs [hunk 6]
```diff
@@ -493,10 +491,10 @@ pkgInfoScopes =
 -- -------------------------------------------------------------------------- --
 -- main
 
--- KILLSWITCH for version 2.2
+-- KILLSWITCH for version 2.3
 --
 killSwitchDate :: Maybe String
-killSwitchDate = Just "2020-11-19T00:00:00Z"
+killSwitchDate = Just "2020-12-22T00:00:00Z"
```
Comment (larskuhtz) [inline node/ChainwebNode.hs:497]:
```suggestion
killSwitchDate = Just "2021-01-14T00:00:00Z"
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,16 @@
...
+## 2.3 (2020-11-11)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-11-19T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2021-01-14T00:00:00Z**.
+
+*   Support for Pact Events (#1157)
+*   Upgrade Pact version 3.7 (#1158)
```
Comment (slpopejoy) [inline CHANGELOG.md:12]:
```suggestion
*   **Upgrade to Pact 3.7.** This includes an API change to the `pact` endpoints for events. For further information see the [Pact readthedocs](https://pact-language.readthedocs.io/en/latest/pact-reference.html#events-1). (#1157,#1158)
```

in other words make it a single bullet

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,16 @@
 # `chainweb-node` Changelog
 
+## 2.3 (2020-11-11)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-11-19T00:00:00Z**. Node administrators must upgrade to this version
```
Comment (mightybyte) [inline CHANGELOG.md:6]: This is correct.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,16 @@
...
+before that date.
+
+This version will stop working on **2021-01-14T00:00:00Z**.
```
Comment (mightybyte) [inline CHANGELOG.md:9]: This is correct.

File: node/ChainwebNode.hs [hunk 10]
```diff
@@ -493,10 +491,10 @@ pkgInfoScopes =
 -- -------------------------------------------------------------------------- --
 -- main
 
--- KILLSWITCH for version 2.2
+-- KILLSWITCH for version 2.3
 --
 killSwitchDate :: Maybe String
-killSwitchDate = Just "2020-11-19T00:00:00Z"
+killSwitchDate = Just "2021-01-14T00:00:00Z"
```
Comment (mightybyte) [inline node/ChainwebNode.hs:497]: This is correct.

**Review by mightybyte — APPROVED** [review]

**Pull request merged**